files/templates/alertmanager/alertmanager.yaml

```diff
@@ -54,7 +54,7 @@ route:
   # Team Atlas Slack (team)
   - receiver: team_atlas_slack
```

Comment:
```suggestion
  - receiver: team_atlas_slack
  matchers:
    [[- if eq .Pipeline "stable" ]]
    - severity!="page"
    [[- end ]]
    - team="atlas"
  continue: false
```

Reply:
That way the testing paging alerts are send to #alert-atlas-test and there is no blackhole

Reply:
We can improve opsgenie to remove the paging alerts also